src/logistration/LoginPage.jsx

```diff
@@ -139,7 +141,7 @@ class LoginPage extends React.Component {
           finishAuthUrl={thirdPartyAuthContext.finishAuthUrl}
         />
         <div className="d-flex justify-content-center login-container">
-          <div className="d-flex flex-column" style={{ width: '400px' }}>
+          <div className="d-flex flex-column login-form-width">
```

Comment:
I have removed these hardcoded widths on all forms in this https://github.com/edx/frontend-app-logistration/pull/69, these are causing issues with the responsiveness of the form on the smaller devices. Kindly remove these changes and rebase with the latest master branch.